packages/synapse-client/out/modelGeneric.mustache

```diff
@@ -6,17 +6,19 @@ import {
     {{classname}}FromJSON,
     {{classname}}FromJSONTyped,
     {{classname}}ToJSON,
+    {{classname}}ToJSONTyped,
```

Comment:
We upgraded the OpenAPI generator version. The OpenAPI generation tool allows you to provide your own template file(s) to customize code generation.

The changes to this file match the upstream changes. Our customizations to this template were unaffected by upstream changes.